src/radical/pilot/agent/executing/popen.py

```diff
@@ -520,9 +523,9 @@ def _check_running(self, to_watch, to_cancel):
     # --------------------------------------------------------------------------
     #
     # pylint: disable=unused-argument
-    def _get_prof(self, event, tid, msg=''):
+    def _get_prof(self, event, tid, msg=None):
 
-        return '$RP_PROF %s\n' % event
+        return '$RP_PROF %s%s\n' % (event, ' %s' % msg if msg else '')
```

Comment:
We probably want to quote the `msg` part, like:

```py
return '$RP_PROF %s%s\n' % (event, ' "%s"' % msg if msg else '')
```

but then this would also work and maybe be a bit simpler?
```py
def _get_prof(self, event, tid, msg=''):
    return '$RP_PROF %s "%s"\n' % (event, msg)
```

Reply:
didn't want to send an empty string, should we skip it or you are in favor to set it anyway?